### PR TITLE
Add consent fields

### PR DESCRIPTION
**Pull request by jamesnicholls**
Adds GDPR consent fields to the B2B prospect form. See https://trello.com/c/OfjQHHaf/391-add-consent-to-channel-barrier.

image

### CODE REVIEW

File: views/form.html [hunk 1]
```diff
@@ -40,20 +40,26 @@ <h2 class="prospect-form__heading">And get 30 more days of free FT</h2>
...
 			{{> n-conversion-forms/partials/phone isB2b=true }}
 
 			{{#unless isUnmasking}}
 			{{> n-conversion-forms/partials/country options=countries }}
 			{{/unless}}
 
-			{{#> n-conversion-forms/partials/fieldset name="accept-terms" legend="Terms and conditions" }}
-			{{#*inline "fields"}}
+			{{#ifAll @root.flags.channelsBarrierConsent consent}}
+				<div>
```
Comment (liamkeaton) [inline views/form.html]: Why the random div?

**Comment by liamkeaton** [comment]
Do we need post in these consent options?

File: views/form.html (hunk 1 above)
Comment (jamesnicholls) [reply views/form.html]: No particular reason, just wanted a single container element (the partial doesn't render one).

**Comment by jamesnicholls** [comment]
@liamkeaton Yeah, based on what I've been told. The consents are stored in the FOW record, so can be changed without a code change anyway.

**Pull request merged**